### DISCUSSION TITLE
Hi,guys,I want to ask why my webpage always prompts for errors？

### DISCUSSION THREAD

**Discussion by Huanxin0722** [lllyasviel/Fooocus#232] [Q&A]
Hi,guys,I want to ask why my webpage always prompts for errors？
''error
Expecting value: line 1 column 1 (char 0)''
I tried to turn off'' clash''.
But after closing, I was prompted that I could not connect.
I've been working on it all night and it's still unsolved. This problem drives me crazy!
[
![图片](https://github.com/lllyasviel/Fooocus/assets/142834042/1ae29f4f-3a47-425c-a862-5bf9c5c11fb9)
](url)

**Comment by mirek190** [comment]
no information about browser , add-ons , system ... how someone can help you ?

**Comment by msaloe** [comment]
I have the same problem. Has this problem been solved?
